portfolio/src/main/webapp/index.html

```diff
@@ -32,18 +32,46 @@
           <button class="tablinks" onclick="openCourses(event, 'music')">Relevant Music coursework</button>
           <button class="tablinks" onclick="openCourses(event, 'research')">Research</button>
         </div>
+        <div class="bg" id="bg-school">
+            <div class="school-div">
+                <span class="school-title">I'm a rising junior at <a href="https://www.mit.edu" id="mit-a">MIT</a> double-majoring in computer science and music.</span>
+                <div class="tab">
+                    <button class="tablinks active" onclick="openCourses(event, 'cs')">Relevant CS coursework</button>
+                    <button class="tablinks" onclick="openCourses(event, 'music')">Relevant Music coursework</button>
+                    <button class="tablinks" onclick="openCourses(event, 'research')">Research</button>
+                </div>
 
-        <!-- Tab content -->
-        <div id="cs" class="tabcontent">
-          <p>CS coursework placeholder</p>
-        </div>
+                <!-- Tab content -->
+                <div id="cs" class="tabcontent">
+                    <p>Design and Analysis of Algorithms</p>
+                    <p>Software Systems for Data Science</p>
+                    <p>Artificial Intelligence</p>
+                    <p>Fundamentals of Programming</p>
+                    <p>Introduction to Machine Learning</p>
+                    <p>Introduction to Algorithms</p>
+                    <p>Oral Communication (for CS majors)</p>
+                    <p>Optimization Methods in Business Analytics</p>
+                    <p>Differential Equations (Recitation Instructor)</p>
+                    <p>Multivariable Calculus<p>
+                </div>
 
-        <div id="music" class="tabcontent">
-          <p>Music coursework placeholder</p>
-        </div>
+                <div id="music" class="tabcontent">
+                    <p>MIT Symphony Orchestra</p>
+                    <p>Harmony and Counterpoint I</p>
+                    <p>Moments in Music: Ragtime Composition</p>
+                    <p>Introduction to World Music</p>
+                </div>
 
-        <div id="research" class="tabcontent">
-          <p>Research placeholder</p>
+                <div id="research" class="tabcontent">
+                    <h5>MIT CSAIL - Vision Group Researcher</h5>
+                    <p>Diagnosed mode collapse of GANs and investigated potential repairs</p>
+                    <p>Implemented model rational methods, including SHAP, LIME, and similarity-based saliency maps, to calculate similarity deltas for Conditional KNN BallTree decisions</p>
```

Comment:
Can you avoid this long line? I know it is optional so you can pass it if you think long line is not avoidable.
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

BTW, are you using any auto format tool?

Reply:
Ah nvw, just saw this branch is removed. ;-)